packages/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.10
```

Comment:
This patch actually updated the response of the public API so it should be 4.1.0